airootfs/usr/share/vx-img/scrape-image.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+IFS=$'\n' read -r -d '' -a disks <<< "$(lsblk -x SIZE -nblo NAME,LABEL,SIZE,TYPE | grep "disk" | awk '{print $1}')"
```

Comment:
is this extraneous as the same command is being called later in the while loop?

Reply:
yep, that's just laziness on my part.